modules/conversion-center/assets/js/editor/module.js

```diff
@@ -0,0 +1,36 @@
+import Component from './component';
+
+class LinksPageLibraryModule extends elementorModules.editor.utils.Module {
+	onElementorLoaded() {
+		this.component = $e.components.register( new Component( { manager: this } ) );
+	}
+
+	onElementorInit() {
+		elementor.hooks.addFilter( 'elements/base/behaviors', ( behaviors ) => {
+			if ( this.isLinksDocument() ) {
+				const { contextMenu: { groups } } = behaviors;
+				behaviors.contextMenu.groups = groups
+					.map( this.filterOutUnsupportedActions() )
+					.filter( ( group ) => group.actions.length );
+			}
+			return behaviors;
+		}, 1000 );
```

Comment:
Why do we need the `1000`?

Reply:
@louiswol94 because there might be other filters added that we would wanto to strip away. it's just to be on the safe side.